st2common/st2common/models/api/trigger.py

```diff
@@ -152,6 +152,10 @@ def to_model(cls, trigger):
                           parameters=parameters)
         return model
 
+    def to_dict(self):
+        # Return dictionary version of the trigger
+        return vars(self)
```

Comment:
Yeah, that's actually an API model and not a DB one so it's less of an issue.

It's also worth nothing that that's pretty much how our API response serialization works right now - calling `json.dumps` on API model works almost in the same manner as that code here :)